server/build.rs

```diff
@@ -20,7 +21,7 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
         .build_client(false)
         .compile(&[proto_file], &[proto_dir])?;
 
-    println!("cargo:rerun-if-changed={}", proto_file);
+    println!("(build) cargo:rerun-if-changed={}", proto_file);
```

Comment:
This should not be prefixed as the line will be parsed by the build process to determine the `rerun-if-changed` configuration

Reply:
https://github.com/Arrow-air/svc-assets/pull/20/commits/660bece8217d0ef1e6813c28c8c1496e123a197d